src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -166,7 +166,7 @@ def create_application_gateway(cmd, application_gateway_name, resource_group_nam
     IPAllocationMethod = cmd.get_models('IPAllocationMethod')
 
     tags = tags or {}
-    sku_tier = sku.split('_', 1)[0]
+    sku_tier = sku.split('_', 1)[0] if 'v2' not in sku else sku
```

Comment:
so this means if `WAF_v2` will be accepted as a valid sku? I thought it should be the opposite.

Reply:
After cross check the sku list, I guess this is correct.